option optimize_for = LITE_RUNTIME;
import "U64Type.proto";
message ArenaStartRequest
{
 required U64Type player_id = 1;
 required uint32 rank = 2;
}
